openApi/openapi.yaml

```diff
@@ -264,9 +264,11 @@ paths:
       operationId: setWalletInstanceStatus
       requestBody:
         content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/SetWalletInstanceStatusData'
+          text/plain:
```

Comment:
please revert to the previous version of this openapi spec (moreover, this is invalid synatx)

see https://github.com/pagopa/io-wallet/blob/master/apps/io-wallet-web-func/openapi.yaml#L45